code/modules/antagonists/brother/brother.dm

```diff
@@ -23,9 +23,35 @@
 	owner.special_role = special_role
 	if(owner.current)
 		give_pinpointer()
+		equip_brother()
 	finalize_brother()
 	return ..()
 
+/datum/antagonist/brother/proc/equip_brother()
+	var/mob/living/carbon/H = owner.current
+	if(!istype(H))
+		return
+	. += brother_give_item(/obj/item/book/granter/crafting_recipe/weapons, H)
+	
+/datum/antagonist/brother/proc/brother_give_item(obj/item/item_path, mob/living/carbon/human/H)
+	var/list/slots = list(
+		"backpack" = SLOT_IN_BACKPACK,
+		"left pocket" = SLOT_L_STORE,
+		"right pocket" = SLOT_R_STORE
+	)
+	
+	var/T = new item_path(H)
+	var/item_name = initial(item_path.name)
+	var/where = H.equip_in_one_of_slots(T, slots)
+	if(!where)
+		to_chat(H, span_userdanger("Unfortunately, you weren't able to get a [item_name]. This is very bad and you should adminhelp immediately (press F1)."))
+		return FALSE
+	else
+		to_chat(H, span_danger("You have a [item_name] in your [where]."))
+		if(where == "backpack")
+			SEND_SIGNAL(H.back, COMSIG_TRY_STORAGE_SHOW, H)
+		return TRUE
+		
```

Comment:
```suggestion
/datum/antagonist/brother/proc/equip_brother()
	var/list/recipes = list(/datum/crafting_recipe/baseball_bat, /datum/crafting_recipe/lance, /datum/crafting_recipe/knifeboxing, /datum/crafting_recipe/flamethrower, /datum/crafting_recipe/pipebomb)
	
	for(var/datum/crafting_recipe/R in recipes)
		owner.teach_crafting_recipe(R)
		to_chat(owner, span_notice("You learned how to make [initial(R.name)]."))
```

Reply:
unless i tested it wrong this doesn't seem to give them the recipes

Reply:
since makeshift book changes you could change this or you could call the makeshift book and it's list (var/obj/item/book/granter/crafting_recipe/weapons/book) -> (var/list/recipes = book.crafting_recipes or whatever the var is called)